crates/cpuid-utils/src/host.rs

```diff
@@ -117,7 +117,16 @@ fn collect_cpuid(
             // Leaf 7 subleaf 0 eax indicates the total number of leaf-7
             // subleaves.
             0x7 => {
-                let data = query(leaf, 0)?;
+                let mut data = query(leaf, 0)?;
+
+                // Leaf 7 EBX bits 12 and 15 indicate PQM and PQE support on AMD
```

Comment:
style nit: I think it would be nice to use the `h` suffix consistently, even for numbers (7, F) which do not require it for disambiguation.

Reply:
fair enough. it turns out we use `0x` prefixes for leaves elsewhere in Propolis anyway, so i've switched to those along the way.